src/main/java/com/balloon_spring_jpa/balloon/exception/CustomerException.java

```diff
@@ -0,0 +1,15 @@
+package com.balloon_spring_jpa.balloon.exception;
+
+import java.util.UUID;
+
+public class CustomerException extends RuntimeException{
+
+    public CustomerException(UUID id) {
+        super("there is no such customer with this id: " + id);
+    }
+
+    @Override
+    public void printStackTrace() {
```

Comment:
You do not have to override the non-abstract method if you don't change/add functionality.